cli/args.go

```diff
@@ -66,7 +66,7 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 		return nil, err
 	}
 
-	sourceUpdate := parseBooleanArg(args, OPT_TERRAGRUNT_SOURCE_UPDATE, false)
+	sourceUpdate := parseBooleanArg(args, OPT_TERRAGRUNT_SOURCE_UPDATE, os.Getenv("TERRAGRUNT_SOURCE_UPDATE") == "true" || os.Getenv("TERRAGRUNT_SOURCE_UPDATE") == "1")
```

Comment:
Just a general comment - I prefer to be lenient about what is accepted when testing Env Vars, eg. in this case, I'd convert the env var value to lower case before comparing so the user can specify "TRUE" or "True" as well as "true".

Reply:
Seems like a good idea. PR to create a helper function for that and apply it everywhere would be very welcome!